SPECS-EXTENDED/linuxptp/linuxptp.spec

```diff
@@ -93,15 +209,47 @@ PATH=..:$PATH ./run
 %{_sbindir}/phc_ctl
 %{_sbindir}/pmc
 %{_sbindir}/ptp4l
-%{_sbindir}/snmp4lptp
 %{_sbindir}/timemaster
+%{_sbindir}/ts2phc
 %{_mandir}/man5/*.5*
 %{_mandir}/man8/*.8*
 
 %changelog
-* Thu Oct 14 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.0-8
-- Initial CBL-Mariner import from Fedora 32 (license: MIT).
-- Converting the 'Release' tag to the '[number].[distribution]' format.
+* Wed Jan 11 2023 Miroslav Lichvar <mlichvar@redhat.com> 3.1.1-6
```

Comment:
We have missed to add a changelog entry.

Reply:
Thanks! That is why the License check is failing. I will work on adding this.